packages/protobuf-21/PKGBUILD

```diff
@@ -56,11 +57,20 @@ build() {
   PROTOC="$srcdir"/build/protoc python -m build --wheel --no-isolation
 }
 
-package() {
+package_protobuf-21() {
   provides=(libprotoc.so libprotobuf.so libprotobuf-lite.so)
   DESTDIR="$pkgdir" cmake --install build
   cd $_name-$pkgver
   install -vDm 644 LICENSE -t "$pkgdir"/usr/share/licenses/$pkgname
   mv "$pkgdir"/usr/bin/protoc{,-21}
   ln -rs "$pkgdir"/usr/lib/$pkgname/*.so.* "$pkgdir"/usr/lib
 }
+
+package_python-protobuf-21() {
+  pkgdesc='Python bindings for Google Protocol Buffers (legacy)'
+  provides=(python-protobuf=$pkgver)
+  depends=(protobuf-21=$pkgver)
+  conflicts=(python-protobuf)
```

Comment:
@SpotlightKid it's not possible to have two versions of the same python module installed at once, right? Or could the legacy version somehow exist under a separate path and how would python know which version to use?

Reply:
No, not system-wide (i.e. in `<prefix>/lib/python<N.N>/site-packages).

If you installed the protobuf 2.1 Python bindings in another directory, you would have to either fix the Python scripts / tests, which should use them, or start with the `PYTHONPATH` environment variable set to the path of that directory (or as the first entry in a list of directories). But that's rather hacky, imho.